src/main/java/csv/PersonCSV.java

```diff
@@ -0,0 +1,84 @@
+package main.java.csv;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.io.Serializable;
+
+public class PersonCSV implements Serializable {
+    private final String name;
+    private final int age;
+    private final String city;
+
+    public PersonCSV(String name, int age, String city) {
+        this.name = name;
+        this.age = age;
+        this.city = city;
+    }
+
+    public String getHeaderCSVRaw() {
+        String headerRaw = "Name,Age,City\n";
+        return headerRaw;
+    }
+
+    public List<PersonCSV> getAllPersons() {
+        List<PersonCSV> persons = new ArrayList<>();
+
+        PersonCSV person1 = new PersonCSV("Maga", 22, "Kyiv");
+        PersonCSV person2 = new PersonCSV("Sasha", 21, "Kyiv");
+        PersonCSV person3 = new PersonCSV("Tima", 63, "Kyiv");
+
+        persons.add(person1);
+        persons.add(person2);
+        persons.add(person3);
+
+        return persons;
+    }
+
+    public String getStringByPerson(PersonCSV person) {
```

Comment:
Maybe, it is better to name it like this: "toCsvRow".
Because as a result we got an CSV row.